package/tests/conftest.py

```diff
@@ -420,10 +420,10 @@ def example_catalog():
 @pytest.fixture
 def example_transcoded_pipelines():
     def process_data(raw_data, train_test_split):
-        ...
+        pass
```

Comment:
Otherwise, the ruff formatter reformats that to one liner:

```python
def process_data(raw_data, train_test_split): ...
```

And `test_task_node_metadata` in `package/tests/test_api/test_rest/test_responses.py` fails. See my modification to that test, to make it compatible with `pass`, below.